R/learner_survivalsvm_surv_svm.R

```diff
@@ -89,37 +102,55 @@ LearnerSurvSVM = R6Class("LearnerSurvSVM",
         predict_types = c("crank", "response"),
         param_set = ps,
         man = "mlr3extralearners::mlr_learners_surv.svm",
-        label = "Support Vector Machine"
+        label = "Survival Support Vector Machine"
       )
     }
   ),
 
   private = list(
     .train = function(task) {
+      pars = self$param_set$get_values(tags = "train")
+
+      # Regularization parameters are defined separately to be tuned more easily
+      gamma = pars$gamma
+      mu = pars$mu
+      type = pars$type
+      if (!is.null(type) && type == "hybrid") {
+        # a vector with two parameters is required when `type` = "hybrid"
+        gamma.mu = c(gamma, mu)
+      } else {
+        gamma.mu = gamma
+      }
+      # remove `gamma` and `mu`
+      pars$gamma = NULL
+      pars$mu = NULL
+
       with_package("survivalsvm", {
         invoke(survivalsvm::survivalsvm,
           formula = task$formula(),
           data = task$data(),
-          .args = self$param_set$get_values(tags = "train"))
+          gamma.mu = gamma.mu, # pass `gamma.mu` separately
+          .args = pars)
       })
     },
 
     .predict = function(task) {
       pars = self$param_set$get_values(tags = "predict")
-      fit = predict(self$model, newdata = ordered_features(task, self),
-        .args = pars
-      )
+      fit = invoke(predict,
```

Comment:
why are there so many indentation changes?
In general we want to change code as little as possible in a meaningful way so the diffs between files actually express relevant changes and not just style changes

Reply:
haha yes - my feeling at the time was since I am looking/fixing/documenting at all these learners, I tried to harmonize also a bit the code, and well as you saw, I have my favorite style :) I will be a bit more reluctant to do this in the future